README.md

```diff
@@ -13,6 +13,44 @@ The CLI is designed for authenticating and returning an access token for public
 | Ubuntu (linux)   | ❌ (not yet) | ✅ | ✅ | ✅ | ❌ (not yet) |
 <br/>
 
+# Installation
+
+## Windows
+
+On Windows we provide a PowerShell bootstrap script, which will download and extract the application to
+`%LOCALAPPDATA%\AzureAuth` and automatically add the `azureauth` binary to your `$PATH`. We don't currently provide a
+means of downloading the latest release, so you **must** specify your desired version via the `$AZUREAUTH_VERSION`
+environment variable.
+
+To install the application, run
+
+```powershell
+# v0.1.0 is just an example here. See https://github.com/AzureAD/microsoft-authentication-cli/releases for the latest.
+$env:AZUREAUTH_VERSION = 'v0.1.0'
+iex "& { $(irm https://raw.githubusercontent.com/AzureAD/microsoft-authentication-cli/main/install/install.ps1) }"
+```
```

Comment:
UX: I'd prefer we split these into 2 separate blocks so that each is copy-able on it's own. Some terminals will warn you about trying to paste multi-lines, and often I don't like to paste multi-lines into my terminal away because I can't change what's happening. It appears you can't add suggestions that span other code blocks but I suggest making the comment normal text, and then making each of these it's own code block.

Reply:
> so that each is copy-able on its own

That's already the case. Nothing forces someone to copy all the lines at once.

> Some terminals will warn you about trying to paste multi-lines

That seems like reasonable behavior

> I can't change what's happening

Unless you're using a bad terminal emulator you usually _can_ change what's happening. Though, I honestly can't speak for `cmd.exe` in this case, but the text is specific to PowerShell anyway.

> I suggest making the comment normal text

I purposefully made the text there a comment because I thought it would increase the likelihood that a dev about to paste code into their terminal would read it, but I'm fine to move it above as well.

I'm happy to make these changes because I'm not really tied to the presentation here, but I'm not convinced that they improve the UX of this document in a significant way.

Reply:
Tucker and talked about this offline and I agreed with his comments, leaving the block as one is the better 90% use case to make this just work for people.